app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -3962,7 +3962,7 @@
 }
 
 //  Configuration -> Payment Methods
-.adminhtml-system-config-edit {
+.adminhtml-system_config-edit {
```

Comment:
Let's keep existing selector for backward compatibility
```suggestion
.adminhtml-system-config-edit,
.adminhtml-system_config-edit {
```

Reply:
thanks, changed it.